SWDEV-310152 - OPENMP specific RUNPATH added to aomp build scripts

raramakr opened the pull request:
RUNPATH in Binaries: $ORIGIN/../lib:$ORIGIN/../../lib
RUNPATH in Libraries: $ORIGIN:$ORIGIN/../../lib

Added the changes to openmp, flang and flang_runtime build scripts

raramakr committed:
SWDEV-310152 - OPENMP specific RUNPATH added to aomp build scripts

RUNPATH in Binaries: $ORIGIN/../lib:$ORIGIN/../../lib
RUNPATH in Libraries: $ORIGIN:$ORIGIN/../../lib

Added the changes to openmp, flang and flang_runtime build scripts

estewart08 reviewed (changes requested):
This needs guarded with AOMP_STANDALONE_BUILD=0. We do not want this in AOMP.

raramakr committed:
SWDEV-310152 - Use OPENMP specific RUNPATH for non standalone builds

raramakr commented:
> This needs guarded with AOMP_STANDALONE_BUILD=0. We do not want this in AOMP.

Updated the patch by addressing the above comments

estewart08 commented:
> > This needs guarded with AOMP_STANDALONE_BUILD=0. We do not want this in AOMP.
> 
> Updated the patch by addressing the above comments

I am thinking we should change the name to something more descriptive as well. Maybe ```ROCM_OPENMP_ORIGIN_RPATH``` or ```OPENMP_EXTRAS_ORIGIN_RPATH```.

raramakr committed:
Updated the variable name for RPATH setting to make it more meaningful

raramakr commented:
> > > This needs guarded with AOMP_STANDALONE_BUILD=0. We do not want this in AOMP.
> > 
> > 
> > Updated the patch by addressing the above comments
> 
> I am thinking we should change the name to something more descriptive as well. Maybe `ROCM_OPENMP_ORIGIN_RPATH` or `OPENMP_EXTRAS_ORIGIN_RPATH`.

Used OPENMP_EXTRAS_ORIGIN_RPATH as variable name for RPATH settings

estewart08 reviewed (approved):
LGTM
